app/assets/javascripts/models/tour.js

```diff
@@ -0,0 +1,51 @@
+import Tether from 'tether';
+import Shepherd from 'tether-shepherd';
+
+module.exports = Backbone.Model.extend({
```

Comment:
Do you think that would be better to move this file to another folder, or just change it to a simple class?